torch/cuda/memory.py

```diff
@@ -467,3 +467,38 @@ def _format_count(cnt, pref_cnt):
     for k, v in stats.items():
         fmt_dict[k.replace(".", "-")] = v
     return "|" + "|\n|".join(lines).format(**fmt_dict) + "|\n"
+
+
+def list_gpu_processes(device: Union[Device, int] = None) -> str:
+    r"""Returns a human-readable printout of the running processes
+    and their GPU memory use for a given device.
+
+    This can be useful to display periodically during training, or when
+    handling out-of-memory exceptions.
+
+    Arguments:
+        device (torch.device or int, optional): selected device. Returns
```

Comment:
nit: I think this could accept a string, too. 

https://github.com/pytorch/pytorch/blob/8d570bc708a63009f3be3f892767bdf4c2ac13a4/torch/cuda/_utils.py#L8